scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/JobState.java

```diff
@@ -438,6 +438,15 @@ public long getCumulatedCoreTime() {
         return getJobInfo().getCumulatedCoreTime();
     }
 
+    /**
+     * To get the numberOfNodes
```

Comment:
Returns the number of nodes used by the job since the beginning of its execution